src/main/java/org/myrobotlab/service/ProgramAB.java

```diff
@@ -1296,6 +1298,16 @@ synchronized public void addCategoryToFile(Bot bot, Category c) {
       error(e);
     }
   }
+  
+  boolean sleep = false;
+  
+  public void wake() {

```

Comment:
are you intending these to be invoked via aiml ?

Reply:
yes - it worky ... tried first with single method and boolean ... and that failed ...   :(